docs/en/01-how-to-build/snpe.md

```diff
@@ -136,7 +136,7 @@ $ cmake .. \
 ### 3) Compile demo
 
 ```bash
-$ cd /path/to/install/example
+$ cd /path/to/install/example/cpp
```

Comment:
We can remove `Compile demo` by enabling `MMDEPLOY_BUILD_EXAMPLES` in `Compile mmdeploy SDK` chapter